src/deadline/nuke_submitter/ui/components/scene_settings_tab.py

```diff
@@ -89,9 +201,27 @@ def refresh_ui(self, settings: RenderSubmitterUISettings):
 
         self.proxy_mode_check.setChecked(settings.is_proxy_mode)
 
+        self.timeout_checkbox.setChecked(settings.timeouts_enabled)
+
+        def _set_timeout(
+            timeout_boxes: tuple[QLabel, QSpinBox, QSpinBox, QSpinBox], timeout_seconds: int
+        ):
+            days = timeout_seconds // 86400
+            hours = (timeout_seconds - days * 86400) // 3600
+            minutes = (timeout_seconds - days * 86400 - hours * 3600) // 60
```

Comment:
alternative: you can use the modulo operator here instead of converting the days and hours back to seconds each time

```python
# 2 days, 3 hours, 7 minutes
>>> ts = (2 * 86400) + (3 * 3600) + (7 * 60)
>>> ts
184020
>>> (ts % 86400) // 3600
3
>>> (ts % 86400 % 3600 ) // 60
7
```

Reply:
Ah, yeah this is way cleaner. Thanks for the suggestion, added!